src/slidy/slidy.js

```diff
@@ -110,12 +106,16 @@ export function slidy (slider, options) {
   }
 
   function _setTailArrowClasses () {
-    if (!updateTailArrows) { return }
+    if (infinite) { return }
     if (prevArrow && prevArrow.classList) {
-      prevArrow.classList[ index < 1 ? 'add' : 'remove' ](tailArrowClass)
+      index < 1
```

Comment:
I was thinking something like that. :)

```
const action = index < 1 ? 'add' : 'remove'
prevArrow.classList[action](tailArrowClass)
```

Main "concern" was more about having a ternary inside, but you can put it directly in a const and it's fine.